<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LuluWinz</class>
 <widget class="QDockWidget" name="LuluWinz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Get PointCloud/Depth/RGB from CAD model</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
      <item>
       <widget class="QPushButton" name="myButton">
        <property name="text">
         <string>Load Scene</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QCheckBox" name="_saveDepth">
          <property name="text">
           <string>Save Depth Image(-s)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="_saveRGB">
          <property name="text">
           <string>Save RGB Image(-s)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="_bottom">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>bottom</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2" alignment="Qt::AlignLeft">
         <widget class="QCheckBox" name="_right">
          <property name="text">
           <string>right</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QCheckBox" name="_right2">
          <property name="text">
           <string>right2</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" alignment="Qt::AlignLeft">
         <widget class="QCheckBox" name="_top">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>top</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="_left">
          <property name="text">
           <string>left</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QCheckBox" name="_left2">
          <property name="text">
           <string>left2</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="_savePointCloud">
          <property name="text">
           <string>Save Point Cloud(-s)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="_showPointCloudTop">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>PC</string>
          </property>
          <property name="iconSize">
           <size>
            <width>3</width>
            <height>3</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="run">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
